src/ontology/mondo-edit.obo

```diff
@@ -277834,6 +277866,7 @@ property_value: confidence "0.924259768009768" xsd:double
 id: MONDO:0014140
 name: muscular dystrophy-dystroglycanopathy (congenital with brain and eye anomalies), type A14
 synonym: "MDDGA14" RELATED ABBREVIATION [MONDO:Lexical, OMIM:615350]
+synonym: "muscle-eye-brain-GMPPB related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/